DEVELOPMENT.md

```diff
@@ -0,0 +1,105 @@
+# Developing
+
+## Getting started
+
+Many of the instructions here replicate what's in the [tektoncd/pipeline development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md), with a couple of caveats currently.
+
+1. This project currently does not support being built with `ko`
+2. This project has not yet been tested with GKE
+
+We would love to accomplish both of these tasks and to update this document, contributions welcome!
+
+### Requirements
+
+You must install these tools:
+
+1. [`go`](https://golang.org/doc/install): The language Tekton Dashboard is
```

Comment:
Don't we need `node` too ? (for react & co)

Reply:
Yep mb, adding the frontend parts next - indeed node and npm for local frontend dev is required 😄

Reply:
I've discussed this with @AlanGreene who works on our frontend, he's kindly agreed to run with this as a new PR. We should get what we have in for now as this gives us room to add the API definitions for our backend next and we'll have something in place at least that lets people know how to build and run the backend. I'm going to call out that the frontend reqs are coming up!